pkg/crc/config/settings.go

```diff
@@ -109,6 +118,9 @@ func RegisterSettings(cfg *Config) {
 
 	cfg.AddSetting(KubeAdminPassword, "", ValidateString, SuccessfullyApplied,
 		"User defined kubeadmin password")
+
+	cfg.AddSetting(PresetConfigurationKey, string(Minimal), validatePreset, RequiresDeleteMsg,
```

Comment:
@guillaumerose here we are considering `minimal` as what we have today not the microshift one right?

Reply:
oh yes good point. should it be `Podman`, `Microshift`, `OpenshiftInstallerCustomized`, `OpenshiftInstallerDefault`?

Reply:
@guillaumerose let's discuss it tomorrow team meeting and then we will make this change.

Reply:
`Podman`, `Microshift` (`MicroShift`?), `OpenShiftMinimal`, `OpenShiftFull`?